compoments/report-repair/contact-details.js

```diff
@@ -50,12 +52,8 @@ const ContactDetails = ({handleChange, values}) => {
   const beforeButton =  (
     <Details summary="I have neither a mobile number nor an email address" testid="no-applicable-contact-options-prompt">
       <div data-testid='no-applicable-contact-options-info'>
-        Please call 01522 873333 to report your repair during the office hours below
-        <li>Monday 9am - 5pm</li>
-        <li>Tuesday 9am - 5pm</li>
-        <li>Wednesday 9am - 4.30pm</li>
-        <li>Thursday 10am - 5pm</li>
-        <li>Friday 9am - 4.30pm</li>
+        Please call {customerServicesTelephoneNumber} to report your repair during the office hours below
+        <OpeningHours />
```

Comment:
If opening hours is defined as a sentence, this doesn't render like the prototype:
![image](https://user-images.githubusercontent.com/1875867/191968677-db3f9c53-84f6-4293-bf2f-3eb87285dbfb.png)

This is in part due to changes in the prototype that haven't been reflected yet and would need to be considered when making those changes.

Reply:
I feel like making these changes here blurs the scope of this ticket. There are a few places where the page content needs to change. For example, the prototype doesn't show `emergency out-of-hours contacts`, but our current code does, and a council may have an emergency out-of-hours number.
 
I feel these content changes should be done as part of those tickets (or, if they don't exist, be created as separate tickets). What do you think?

Reply:
I guess that if we don't do them here, we can create a new ticket & link it. I agree with @Renny4Real that it seems out of scope for this ticket.

Fyi, it seems that the blue text is different too.